gateway/ln-gateway/src/rpc/rpc_client.rs

```diff
@@ -109,6 +110,41 @@ impl GatewayRpcClient {
         self.call_post(url, payload).await
     }
 
+    pub async fn connect_to_peer(&self, payload: ConnectToPeerPayload) -> GatewayRpcResult<()> {
+        let url = self
+            .base_url
+            .join("/connect_to_peer")
```

Comment:
Nit: we should probably make constants out of `connect_to_peer` similar to how we've done for the federation API. Perhaps could be a follow up

Reply:
Conforming to existing pattern for now. Created #4545 to track this.